gbfs-validator/nonSchemaValidation/README.md

```diff
@@ -0,0 +1,140 @@
+# last_updated_future
+
+If `last_updated` is in the future, the validator will return an error.
+
+# last_updated_outdated
+
+If `last_updated` is older than 5 minutes for near real-time feeds,
+the validator will return an error.
+
+# additional_properties
+
+If any file contains additional properties, the validator will return a warning, to help detects typos or fields that could be updated to conform to the GBFS specification.
+
+# missing_translation
+
+If any `Localized String` is missing a lang declared in the `languages` field of `system_information.json`, the validator will return an error.
+
+# duplicate_translation
+
+If any `Localized String` has more than one translation for the same lang, the validator will return an error.
+
+# unknown_language
+
+If any `Localized String` has a lang that is not declared in the `languages` field of `system_information.json`, the validator will return an error.
+
+# missing_system_pricing_plans
+
+If a pricing is used (like in `default_pricing_plan_id` or `pricing_plan_ids` of the `vehicle_types.json` file ), but not pricing are declared, the validator will return an error.
+
+# missing_default_pricing_plan_id
+
+If the `default_pricing_plan_id` is not present in the `vehicle_types.json` file when pricing plans are present, the validator will return an error.
+
+# missing_station_information
+
+If a station is declared `station_status.json` but not in `station_information.json`, the validator will return an error.
+
+# missing_station_status
+
+If a station is declared `station_information.json` but not in `station_status.json`, the validator will return an error.
+
```

Comment:
May we add the key `duplicate_station_id` used in `nonSchemaValidation/check_stations.js` but missing from `README.md`?

```suggestion

# duplicate_station_id
If `station_information.json` or `station_status.json` contain duplicate `station_id`, the validator will return an error.

```